dist/index.html

```diff
@@ -5,10 +5,56 @@
     <title>Activity Tracker</title>
   </head>
   <body>
-    <h1>Activity Tracker</h1>
-    <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
 
-<!-- Do not include the scripts.js file here - it is done automatically by the webpack server -->
```

Comment:
I think we may want to keep this so that we don't forget to not include any other files here.